nmosconnection/abstractDevice.py

```diff
@@ -121,8 +120,8 @@ def getStagedParameter(self, parameter, leg=0):
 
     def setActiveParameter(self, value, parameter, leg=0):
         """Set the value of an active parameter"""
-        if parmaeter in self.active[__tp__][leg]:
-            self.staged[__tp__][leg][parameter]
+        if parameter in self.active[__tp__][leg]:
+            self.active[__tp__][leg][parameter] = value
```

Comment:
Huh, wonder how that managed to not do anything all these years. I have a nagging feeling it's because no current implementations actually use this method.